src/components/editable-title/index.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import styled from 'styled-components';
+
+import { Color } from '../colors';
+import { CopySize } from '../copy';
+import { getSpace, SpaceSize } from '../space';
+
+export enum EditState {
+	Editable = 'Editable',
+	Editing = 'Editing'
+}
+
+export interface EditableTitleProps {
+	fontSize?: CopySize;
+	focused: boolean;
+	name: string;
+	nameState: EditState;
+	secondary?: boolean;
+	value: string;
+	onBlur?: React.FocusEventHandler<HTMLInputElement>;
+	onChange?: React.ChangeEventHandler<HTMLInputElement>;
+	onClick?: React.MouseEventHandler<HTMLElement>;
+	onFocus?: React.FocusEventHandler<HTMLInputElement>;
+	onKeyDown?: React.KeyboardEventHandler<HTMLInputElement>;
+}
+
+interface EditableInputProps {
+	autoFocus: boolean;
+	autoSelect: boolean;
+	fontSize?: CopySize;
+	secondary?: boolean;
+	value: string;
+	onBlur?: React.FocusEventHandler<HTMLInputElement>;
+	onChange?: React.ChangeEventHandler<HTMLInputElement>;
+	onFocus?: React.FocusEventHandler<HTMLInputElement>;
+	onKeyDown?: React.KeyboardEventHandler<HTMLInputElement>;
+}
+
+interface StyledEditableTitleProps {
+	children: React.ReactNode;
+	editable: boolean;
+	fontSize?: CopySize;
+	focused?: boolean;
+	secondary?: boolean;
+}
+
+interface StyledInputProps {
+	fontSize?: CopySize;
+	secondary?: boolean;
+}
+
+const StyledTitle = (props: StyledEditableTitleProps): JSX.Element => {
+	const Strong = styled.strong`
```

Comment:
Remove `StyledTitle` and use `Strong` => `StyledStrong` directly.

You can add `data-title` via

```
const StyledStrong = styled.strong.attrs({ "data-title": true })``
```